terraform/logging.tf

```diff
@@ -34,3 +34,9 @@ resource "google_logging_project_sink" "metrics" {
   EOF
 
 }
+
+resource "google_project_iam_member" "metric_viewers" {
```

Comment:
Style question: I'm not sure if it would be better to refactor this out of the module or not, since its a project-level setting.

Reply:
OK, left it, made the variable description a bit more clear.